Mollier/SAM.Core.Mollier.UI/Query/SaveAsPDF.cs

```diff
@@ -6,11 +6,10 @@ namespace SAM.Core.Mollier.UI
 {
     public static partial class Query
     {
-        public static void SaveAsPDF(Chart MollierChart, string path, int fontSize, string size, int newWidth, int newHeight, int chartWidth, int chartHeight)
+        public static void SaveAsPDF(Chart MollierChart, int fontSize, string size, int newWidth, int newHeight, int chartWidth, int chartHeight)
```

Comment:
Add "this" before Chart. Is chart MollierChart? input name should start from small letter